src/pybind/mgr/dashboard/frontend/src/app/shared/enum/notification-type.enum.ts

```diff
@@ -1,5 +1,6 @@
 export enum NotificationType {
   error,
   info,
-  success
+  success,
+  warning
 }
```

Comment:
export enum NotificationType {
  error = "error",
  info = "info",
  success = "success",
  warning = "warning"
}

Lets have this like this, and reuse this elsewhere, I see we are maintaining maps as well which looks redundant and confusing